app/views/summary/updateRegisteredDetails/WarehouseDetailsSummary.scala

```diff
@@ -54,7 +54,7 @@ object WarehouseDetailsSummary  {
                 classes = "govuk-!-font-weight-regular govuk-!-width-full"
               ),
               actions = Some(Actions("",Seq(
-                ActionItemViewModel("site.remove", routes.WarehouseDetailsController.onPageLoad(NormalMode).url)
+                ActionItemViewModel("site.remove", controllers.updateRegisteredDetails.routes.RemoveWarehouseDetailsController.onPageLoad(NormalMode, warehouse._1).url )
```

Comment:
This is navigation to the remove page.  
The ticket should be navigating from the remove page - currently it is going back to index, not warehouse-details